examples/vector-widget/VectorWidget.js

```diff
@@ -8,93 +8,72 @@
  */
 "use strict";
 
-var React = require('react');
-var ReactART = require('react-art');
-var Group = ReactART.Group;
-var Shape = ReactART.Shape;
-var Surface = ReactART.Surface;
-var Transform = ReactART.Transform;
-
-var MOUSE_UP_DRAG = 0.978;
-var MOUSE_DOWN_DRAG = 0.9;
-var MAX_VEL = 11;
-var CLICK_ACCEL = 3;
-var BASE_VEL = 0.15;
+const React = require('react');
```

Comment:
Sure. This file was causing some console warnings before:

> Warning: bind(): React component methods may only be bound to the component instance. See VectorWidget

No changes were _required_ though. I just fixed the warnings and added a little better react-art component coverage. I can do both via a separate PR.